### PR TITLE
feat(podcast): Add podcast iframe block

### PR DESCRIPTION
**Pull request by nimithshetty17**
Issue:-
Fixes https://jira.wh-sdlc.watson-health.ibm.com/browse/MERATIVE-742

Description:-

This PR is for creating a new block to embed the apple and spotify podcast.

Design Specs
Figma Link - https://www.figma.com/file/dFJwrp6VkUMSirF2KMLRMx/Thought-leadership-library?type=design&node-id=357-17001&mode=design&t=NJAN5HiO1NaRacYg-0

Test URLs

After (Changes from this PR): https://podcast-iframe-block--merative2--nimithshetty17.hlx.page/drafts/nimith/podcast
Testing Instruction
Visit one of the documents(eg. https://merative.sharepoint.com/:w:/r/sites/Merative.comWebsite/_layouts/15/doc2.aspx?sourcedoc=%7B6A71CB0E-7486-47E2-82E0-47850F2159B1%7D&file=podcast.docx&action=default&mobileredirect=true)
Add the block by adding the url and the spotify or apple link.

### CODE REVIEW

File: scripts/delayed.js [hunk 1]
```diff
@@ -49,6 +49,15 @@ if (document.querySelector('[data-ytid]')) {
   });
 }
```
Comment (proeung) [inline scripts/delayed.js:51]: Nitpicking: Let's add a comment explaining what this check does.

```suggestion

// Check if the element with id 'podcast-container-id' exists in the document
```

**Review by proeung — CHANGES_REQUESTED** [review]
@nimithshetty17 Code looks good overall! Please see my feedback below.
 
**Needs fixing**
- The podcast iframe embeds are not loading and this is due to the CSP. Please add/allow Spotify and Apple Podcast to the list.
- Example of an article test page - https://podcast-iframe-block--merative2--nimithshetty17.hlx.page/drafts/putra/test-blog-detail-page

Screen Shot 2023-06-22 at 10 04 52 AM

**Feedback/reminder for the next PR**

- I noticed that you're not using [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/#examples). This might be because you haven't updated your package dependencies, which includes changes from this PR (https://github.com/hlxsites/merative2/pull/212/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519R39-R40).. Please make sure to run `npm install` in your local build.
- Also, make sure for your next PR that you're using the PR template format. See (https://github.com/hlxsites/merative2/pull/243) for an example.

**Comment by nimithshetty17** [comment]
@proeung - Thanks for the comments! Have made the changes for the CSP and the podcast is available now for review.

**Comment by sachinmesh** [comment]
Hi @nimithshetty17 @proeung 
checked the functionality in all breakpoints,  working fine. We don't have designs specially created for podcast in design system. We can add the podcast iframe below the speakers title and name above the bodycopy of the podcasts description.

**Review by proeung — APPROVED** [review]
Looks good! I no longer see the CSP errors in the console.

Also, since this "Podcast" is a new block, we'll need to add this to the Franklin Library dialog. We can hold off on making this be available to Content Authors until the changes in this feature branch `feat/thought-leadership` is ready to merge into `main`. @nimithshetty17 @keith-kaplan

**Pull request merged**